source/js/openstreetmap/zoomMarkerScroll.ts

```diff
@@ -0,0 +1,43 @@
+import { Map as LeafletMap, Marker, MarkerClusterGroup } from 'leaflet';
+
+class ZoomMarkerSroll {
+    map: LeafletMap;
+    markers: MarkerClusterGroup;
+    container: Element;
+    scrollToMarkerItems: Array<Element>;
+
+    constructor(map: LeafletMap, markers: MarkerClusterGroup, container: Element) {
+        this.scrollToMarkerItems = [...container.querySelectorAll('[js-scroll-to-marker]')];
+        this.map = map;
+        this.markers = markers;
+        this.container = container;
+
+        this.init();
+    }
+
+    init() {
+        if (this.scrollToMarkerItems.length <= 0) return;
+        
+        this.setListeners();
+        // console.log(this.scrollToMarkerItems);
```

Comment:
```suggestion
```

Reply:
This class doesnt get initiated at the moment. Within short we will add scroll functionality to the map, this class is set up as a test to handle this